apis/dataplane/v1beta1/openstackdataplanenodeset_webhook.go

```diff
@@ -161,6 +164,25 @@ func (r *OpenStackDataPlaneNodeSet) validateNodes() (field.ErrorList, error) {
 
 }
 
+// duplicateMountCheck checks the NodeSet for duplicate mountPaths.
+func (r *OpenStackDataPlaneNodeSet) duplicateMountCheck(extraMount storage.VolMounts) field.ErrorList {
```

Comment:
If you're passing `extraMount` as a parameter, you don't need the receiver, it can just be a function and not a method.

Reply:
True, I originally implemented it as a standalone check, but I realized that we were already iterating over extraMounts, so I could save us a loop by just inserting this check in there with that logic.